lessons/json-in-postgresql.md

```diff
@@ -12,7 +12,7 @@ This course incorrectly chooses to use the `JSON` data type when it should have
 
 [For more info read this blog post about it][jsonb].
 
-## JSON
```

Comment:
Ignore this change please, there is a mistake on my part and I'm trying to delete it.